localstack/services/awslambda/invocation/lambda_models.py

```diff
@@ -64,6 +64,9 @@
 SNAP_START_SUPPORTED_RUNTIMES = [Runtime.java11, Runtime.java17]
 
 
+BUCKET_ACCOUNT = "949334387222"
```

Comment:
✅ We don't get any conflicts in multi-account setups because the `account_id` is used as namespace